LicenseHeaderManager/PackageCommands/RemoveHeaderFromAllFilesInProjectCommand.cs

```diff
@@ -17,11 +17,11 @@
 
 namespace LicenseHeaderManager.PackageCommands
 {
-  class RemoveLicenseHeaderFromAllFilesCommand
+  public class RemoveHeaderFromAllFilesInProjectCommand
   {
     private LicenseHeaderReplacer licenseReplacer;
 
-    public RemoveLicenseHeaderFromAllFilesCommand(LicenseHeaderReplacer licenseReplacer)
+    public RemoveHeaderFromAllFilesInProjectCommand(LicenseHeaderReplacer licenseReplacer)
     {
       this.licenseReplacer = licenseReplacer;
```

Comment:
Again, mistake by me... i am going to clean this up myself after the pull request